src/bootstrap/bootstrap.py

```diff
@@ -197,8 +197,6 @@ def run(args, verbose=False, exception=False, is_bootstrap=False, help_on_error=
     code = ret.wait()
     if code != 0:
         err = "failed to run: " + ' '.join(args)
-        if help_on_error is not None:
-            err += "\n" + help_on_error
         if verbose or exception:
```

Comment:
Please also remove this from the function parameters (for run, and for all functions which call it, recursively)